src/utils/exceptions.ts

```diff
@@ -0,0 +1,92 @@
+import { MongoServerError } from "mongodb";
```

Comment:
hmm I believe the backend refactoring done a few days ago removed this file and moved all exceptions. Could you move each of these errors into their respective `src/utils/exceptions` folder (i think some are duplicates, so just make sure the imports are to the individual folders and not this file) and then remove this file?